diff --git a/types/index.d.ts b/types/index.d.ts
index d66f0cef5e7401292e40d36b983a1abd9305afc4..8d8c5db57dd6d29d6508f30fda3e2faa41384475 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -1,4 +1,4 @@
-import { ComponentType, ElementType } from 'react'
+import type { ComponentType, ElementType, JSX as ReactJSX } from 'react'
 import { Config as TailwindConfig } from 'tailwindcss'
 
 export interface TwStyle {
@@ -25,12 +25,12 @@ export type ScreenFn = <T = string>(
   screenValue: string | TemplateStringsArray | string[]
 ) => (styles?: string | TemplateStringsArray | TwStyle | TwStyle[]) => T
 
-export type TwComponent<K extends keyof JSX.IntrinsicElements> = (
-  props: JSX.IntrinsicElements[K] & { as?: ElementType }
-) => JSX.Element
+export type TwComponent<K extends keyof ReactJSX.IntrinsicElements> = (
+  props: ReactJSX.IntrinsicElements[K] & { as?: ElementType }
+) => ReactJSX.Element
 
 export type TwComponentMap = {
-  [K in keyof JSX.IntrinsicElements]: TemplateFn<TwComponent<K>>
+  [K in keyof ReactJSX.IntrinsicElements]: TemplateFn<TwComponent<K>>
 }
 
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
@@ -48,9 +48,6 @@ declare module 'react' {
   interface HTMLAttributes<T> extends AriaAttributes, DOMAttributes<T> {
     cs?: string
   }
-}
-
-declare global {
   namespace JSX {
     interface IntrinsicAttributes {
       tw?: string
